python/py-lazy_loader/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        scientific-python lazy_loader 0.3
+revision            0
+name                py-${github.project}
```

Comment:
is there a strong need to use the `github` PG? We prefer to fetch Python ports from PyPI, so if possible please switch to that.